test/mocks/data/geopoint.dart

```diff
@@ -68,28 +76,29 @@ class GeoPointGenerator {
 
     const maxDiagonalDistanceInRange = 0.0005;
     const minDiagonalOutRange = 0.001;
-    double distanceBetweenPosts = maxDiagonalDistanceInRange / (inRange + 1);
+    double distanceBetweenpositions =
+        maxDiagonalDistanceInRange / (inRange + 1);
 
     double userLatitude = userPosition.latitude;
     double userLongitude = userPosition.longitude;
 
-    final postInRange = List.generate(inRange, (i) {
-      double dDirection = distanceBetweenPosts * i;
+    final positionInRange = List.generate(inRange, (i) {
+      double dDirection = distanceBetweenpositions * i;
       return GeoPoint(
         userLatitude + dDirection,
         userLongitude + dDirection,
       );
     });
 
-    // Generate post positions that are not in the range.
-    final postsNotInRange = List.generate(outRange, (i) {
-      double dDirection = minDiagonalOutRange + distanceBetweenPosts * i;
+    // Generate position positions that are not in the range.
```

Comment:
I think that there is a repetition here. 
```suggestion
    // Generate positions that are not in the range.
```